src/codegen/structs.ts

```diff
@@ -200,13 +200,17 @@ function findFieldStructTypeString(
 ): string {
     const fieldType = findInternalType(typeString, namespace, abi)
 
-    if (['String', 'Boolean', 'Number'].includes(fieldType)) {
-        return fieldType.toLowerCase()
-    }
-
     if (fieldType === 'Symbol') {
         return 'Asset.Symbol'
     }
 
+    if (fieldType === 'Bool') {
+        return 'boolean'
+    }
+
+    if (['String', 'Boolean', 'Number'].includes(fieldType)) {
```

Comment:
Nice catch here, but if `Boolean` isn't a thing then we should probably remove it from this line.

Reply:
I'm not sure if it is or isn't a thing, so I didn't want to remove it 😅